client/extensions/woocommerce/app/settings/shipping/shipping-zone-list.js

```diff
@@ -0,0 +1,79 @@
+/**
+ * External dependencies
+ */
+import React, { Component } from 'react';
+import { localize } from 'i18n-calypso';
+
+/**
+ * Internal dependencies
+ */
+import Button from 'components/button';
+import Card from 'components/card';
+import ExtendedHeader from '../../../components/extended-header';
+import ShippingZone from './shipping-zone';
+import ShippingZoneDialog from './shipping-zone-dialog';
+
+class ShippingZoneList extends Component {
+	constructor( props ) {
+		super( props );
+
+		//TODO: use redux state with real data
+		this.state = {
+			showDialog: false,
+			shippingZones: [ {
+				locationName: 'United States',
+				locationDescription: '50 states',
+				methodName: 'USPS',
+				methodDescription: 'All domestic services',
+				icon: 'location'
+			}, {
+				locationName: 'Rest of the world',
+				locationDescription: '240 countries',
+				methodName: 'USPS',
+				methodDescription: 'All international services',
+				icon: 'globe'
+			} ],
+		};
+
+		this.onAddZoneOpen = this.onAddZoneOpen.bind( this );
+		this.onAddZoneClose = this.onAddZoneClose.bind( this );
+	}
+
+	onAddZoneOpen() {
+		this.setState( { showDialog: true } );
+	}
+
+	onAddZoneClose() {
+		this.setState( { showDialog: false } );
+	}
+
+	renderShippingZone( zone, index ) {
+		return ( <ShippingZone key={ index } { ...zone } /> );
+	}
+
+	render() {
+		const { translate } = this.props;
+
+		return (
+			<div>
+				<ExtendedHeader
+					label={ translate( 'Shipping Zones' ) }
+					description={ translate( 'The regions you ship to and the methods you will provide.' ) }>
+					<Button onClick={ this.onAddZoneOpen }>{ translate( 'Add zone' ) }</Button>
```

Comment:
Hi! I've found a possible matching string that has already been translated **9** times:
`translate( 'Add DNS Zone' )` *ES Score: 9*
See 1 additional suggestion in the [PR translation status page](https://translate.wordpress.com/localci/status/automattic/wp-calypso/add/13559-shipping-ui)

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).